### PR TITLE
doc: add meeting minutes

### LINKED ISSUE #28: Node.js  Performance Team Meeting 2022-11-28
## Time

**UTC Mon 28-Nov-2022 18:00 (06:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Mon 28-Nov-2022 10:00 (10:00 AM) |
| US / Mountain | Mon 28-Nov-2022 11:00 (11:00 AM) |
| US / Central  | Mon 28-Nov-2022 12:00 (12:00 PM) |
| US / Eastern  | Mon 28-Nov-2022 13:00 (01:00 PM) |
| EU / Western  | Mon 28-Nov-2022 18:00 (06:00 PM) |
| EU / Central  | Mon 28-Nov-2022 19:00 (07:00 PM) |
| EU / Eastern  | Mon 28-Nov-2022 20:00 (08:00 PM) |
| Moscow        | Mon 28-Nov-2022 21:00 (09:00 PM) |
| Chennai       | Mon 28-Nov-2022 23:30 (11:30 PM) |
| Hangzhou      | Tue 29-Nov-2022 02:00 (02:00 AM) |
| Tokyo         | Tue 29-Nov-2022 03:00 (03:00 AM) |
| Sydney        | Tue 29-Nov-2022 05:00 (05:00 AM) |

Or in your local time:
* https://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Performance%20Team+Meeting+2022-11-28&iso=20221128T1800
* or https://www.wolframalpha.com/input/?i=06PM+UTC%2C+Nov+28%2C+2022+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1Yp4arNnwxTO6rv8Pb3RygHcoWnQT-vhD7okUAhKOC98/edit>

## Agenda

Extracted from **performance-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/performance

* Add support for V8 fast call [#23](https://github.com/nodejs/performance/issues/23)
* Merging Github team with README [#19](https://github.com/nodejs/performance/issues/19)
* Buffer.toString vs TextDecoder [#18](https://github.com/nodejs/performance/issues/18)
* fetch [#11](https://github.com/nodejs/performance/issues/11)
* 64 bit integers [#10](https://github.com/nodejs/performance/issues/10)
* WHATWG Streams [#9](https://github.com/nodejs/performance/issues/9)
* Atomic condition variable [#5](https://github.com/nodejs/performance/issues/5)
* Buffer.indexOf [#4](https://github.com/nodejs/performance/issues/4)
* Buffer.(read|write)(U)Int(LE|BE)(8|16|32) [#2](https://github.com/nodejs/performance/issues/2)

## Invited

* Performance team: @nodejs/performance

## Notes

The agenda comes from issues labelled with `performance-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

* link for participants: https://zoom.us/j/94070553680

### PR DESCRIPTION
**Pull request by sheplu**
closes #28

### CODE REVIEW

File: meetings/2022-11-28.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	* We will have a special discussion with multiple people around this topic. If you are interested you can add yourself on the issue 
+* Atomic condition variable [#5](https://github.com/nodejs/performance/issues/5)
+* Buffer.indexOf [#4](https://github.com/nodejs/performance/issues/4)
+	* to be closed
+* Buffer.(read|write)(U)Int(LE|BE)(8|16|32) [#2](https://github.com/nodejs/performance/issues/2)
+	* nothing new
+
+
+## Q&A, Other
+* Yagiz: better version of array-join - could be nice to check  https://github.com/anonrig/fast-array-join
```
Comment (anonrig) [inline meetings/2022-11-28.md:52]:
```suggestion
* Yagiz: Open sourced @BridgeAR's faster version of array-join - could be nice to check  https://github.com/anonrig/fast-array-join
```

File: meetings/2022-11-28.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	* Jean: will try to keep a look at it
+* fetch [#11](https://github.com/nodejs/performance/issues/11)
+	* Ethan: streams are slow by default. Need to find a way to have faster stream (for Fetch but also a lot of other parts of node). Micro optimization is maybe not the best use case here (based on how it is used) and would be better to have one good optimization (based on time passed) 
+    * spec => https://fetch.spec.whatwg.org/
+    * Yagiz : Check turbofan for optimization
+* 64 bit integers [#10](https://github.com/nodejs/performance/issues/10)
+	* nothing new
+* WHATWG Streams [#9](https://github.com/nodejs/performance/issues/9)
+	* We will have a special discussion with multiple people around this topic. If you are interested you can add yourself on the issue 
+* Atomic condition variable [#5](https://github.com/nodejs/performance/issues/5)
```
Comment (anonrig) [inline meetings/2022-11-28.md:44]:
```suggestion
* Atomic condition variable [#5](https://github.com/nodejs/performance/issues/5)
  * removing agenda label due to missing benchmarks
```

File: meetings/2022-11-28.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+* Buffer.indexOf [#4](https://github.com/nodejs/performance/issues/4)
+	* to be closed
```
Comment (anonrig) [inline meetings/2022-11-28.md:46]:
```suggestion
	* Tony did a fantastic job, but unfortunately there isn't any room for improvement at the moment.
```

File: meetings/2022-11-28.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+* Buffer.(read|write)(U)Int(LE|BE)(8|16|32) [#2](https://github.com/nodejs/performance/issues/2)
+	* nothing new
```
Comment (anonrig) [inline meetings/2022-11-28.md:48]:
```suggestion
	* nothing new, moving to the next meetings agenda
```

File: meetings/2022-11-28.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+## Announcements
+
+*Extracted from **performance-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/performance
+
+* Add support for V8 fast call [#23](https://github.com/nodejs/performance/issues/23)
+	* Multiple people are keen to help (Matteo, James, …)
+	* stephen: V8 fast call API can be (very) limited but we need to investigated the gain / impact
+	* Yagiz: find more information and update the PR
```
Comment (anonrig) [inline meetings/2022-11-28.md:30]:
```suggestion
	* Juan & Yagiz: find more information and come up with a PR
```

File: meetings/2022-11-28.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
+# Node.js  Performance Team Meeting 2022-11-28
+
+## Links
+
+* **Recording**:  https://www.youtube.com/watch?v=f0yEem50VOI
+* **GitHub Issue**: $GITHUB_ISSUE$
```
Comment (anonrig) [inline meetings/2022-11-28.md:6]:
```suggestion
* **GitHub Issue**: https://github.com/nodejs/performance/issues/28
```

File: meetings/2022-11-28.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+* Performance team: @nodejs/performance
+* Yagiz Nizipli (@anonrig)
+* Ulises Gascon (@ulisesgascon)
+* Jean Burellier (@sheplu)
+* Ethan Arrowood (@Ethan-Arrowood)
+* Stephen Belanger (@qard)
+* Juan Arboleda (@juanarbol)
+* Marco Ippolito (@marco-ippolito)
+
```
Comment (anonrig) [inline meetings/2022-11-28.md:18]: Stephen is missing from this list afaik

**Review by anonrig — CHANGES_REQUESTED** [review]

**Review by anonrig — APPROVED** [review]

**Review by UlisesGascon — APPROVED** [review]

**Pull request merged**